src/HDInsight/HDInsight/help/Get-AzHDInsightMonitoring.md

```diff
@@ -1,70 +1,70 @@
-﻿---
+---
 external help file: Microsoft.Azure.PowerShell.Cmdlets.HDInsight.dll-Help.xml
 Module Name: Az.HDInsight
-online version: https://docs.microsoft.com/en-us/powershell/module/az.hdinsight/get-azhdinsightoperationsmanagementsuite
+online version: https://docs.microsoft.com/en-us/powershell/module/az.hdinsight/get-azhdinsightmonitoring
 schema: 2.0.0
 ---
 
-# Get-AzHDInsightOperationsManagementSuite
+# Get-AzHDInsightMonitoring
 
 ## SYNOPSIS
-Gets the status of Operations Management Suite (OMS) installation on the cluster.
+Gets the status of monitoring installation on the cluster.
 
 ## SYNTAX
 
 ```
-Get-AzHDInsightOperationsManagementSuite [-Name] <String> [-ResourceGroupName <String>]
+Get-AzHDInsightMonitoring [-Name] <String> [-ResourceGroupName <String>]
  [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Get-AzHDInsightOperationsManagementSuite** cmdlet gets the status of OMS installation in an Azure HDInsight cluster. If OMS is enabled then it will also return the OMS workspace id.
+The **Get-AzHDInsightMonitoring** cmdlet gets the status of monitoring installation in an Azure HDInsight cluster. If monitoring is enabled then it will also return the log analytics workspace id.
 
 ## EXAMPLES
 
 ### Example 1
 ```
-PS C:\> Get-AzHDInsightOMS -Name testcluster
+PS C:\> Get-AzHDInsightMonitoring -Name testcluster
 
 ClusterMonitoringEnabled
```

Comment:
@aim-for-better please make sure the output is updated in the examples

Reply:
> @aim-for-better please make sure the output is updated in the examples

updated